habitat_baselines/rl/ddppo/policy/resnet_policy.py

```diff
@@ -535,6 +560,11 @@ def forward(
             goal_output = self.goal_visual_encoder({"rgb": goal_image})
             x.append(self.goal_visual_fc(goal_output))
 
+        if InstanceImageGoalSensor.cls_uuid in observations:
```

Comment:
Same here:
```
for uuid in [InstanceImageGoalSensor.cls_uuid, ImageGoalSensor.cls_uuid]:
     if uuid in observations:
      ...

Reply:
@mathfac I originally didn't do a loop because keeping the encoders separate would require setting class attributes dynamically. I pushed a commit 4778a46230efbb3a655a4738baf2e43304fbaab5 that does this. Is this what you had in mind?